shopfloor/services/single_pack_transfer.py

```diff
@@ -283,6 +283,12 @@ def cancel(self, package_level_id):
             return self._response_for_start(message=self.msg_store.already_done())
 
         package_level.is_done = False
+        if package_level.picking_id.create_uid == self.env.user:
```

Comment:
and where is the else mark picking as unpicked like you just did on the other scenario in https://github.com/OCA/wms/pull/748 ?